docs/reference/examples/exploitation.json

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "ssvc",
+  "version": "1.0.0",
+  "name": "Exploitation",
+  "description": "The present state of exploitation of the vulnerability.",
+  "key": "E",
+  "values": [
+    {
+      "name": "None",
+      "description": "There is no evidence of active exploitation and no public proof of concept (PoC) of how to exploit the vulnerability.",
+      "key": "N"
+    },
+    {
+      "name": "PoC",
```

Comment:
Maybe a separate issue -- Should we re-name this to "Public PoC" to emphasize it's not just a PoC used to validate that the vulnerability is a real vulnerability?

Reply:
Separate issue.